src/client/modules/subscription/containers/UpdateCard.js

```diff
@@ -12,14 +11,6 @@ import settings from '../../../../../settings';
 
 // react-stripe-elements will not render on the server.
 class UpdateCard extends React.Component {
-  static defaultProps = {
-    updateCard: () => {}
-  };
-
-  static propTypes = {
```

Comment:
Here you removed propTypes also?

Reply:
Yes because the `propTypes` wasn't used in the component.

Reply:
I is used in `UpdateCardWithApollo`, but I guess it does not count for there, since it is outside of `UpdateCard`.

Reply:
I think we discussed this before and queries wrapped under `compose` does not need to have `PropTypes` defined

Reply:
Yes